README.md

```diff
@@ -147,7 +147,7 @@ someFunctionWithABunchOfArguments(
     })
 ```
 
-* **1.12** Prefer using local properties or other mitigation techniques to avoid multi-line predicates where possible.
+* **1.12** Prefer using local variables or constants or other mitigation techniques to avoid multi-line predicates where possible.
```

Comment:
super-minor: I'd put constant first ("constants or variables") or maybe even _just_ "constants" here. shouldn't have to use variables for broken-up predicates I think.